behavior_tree_msgs/srv/SetKvPairs.srv

```diff
@@ -0,0 +1,3 @@
+behavior_tree_msgs/KeyValue[] kv_pairs
+---
+bool received
```

Comment:
Maybe instead of `received` it should be more about if they were set successfully, e,g, `success` or `result` or `updated`.

In addition, we could report which key-value pairs were **not** set successfully in the event of an error. Perhaps as string list:

```
string[] keys_failed
```